### DISCUSSION TITLE
Which libraries does Poetry use for CLI?

### DISCUSSION THREAD

**Discussion by tildebox** [python-poetry/poetry#9709] [Q&A]
After installing Poetry in Windows, you can easily enter the following lines via Terminal / Command Prompt / PowerShell:

> poetry init
> poetry add requests pendulum
> poetry show

Which libraries does Poetry use to be able to do things like poetry...?

**Comment by radoering** [comment]
https://github.com/python-poetry/cleo

**Reply by tildebox** [reply]
Thank you.
